wakamiti-plugins/wakamiti-email/src/main/java/es/iti/wakamiti/email/EmailStepContributor.java

```diff
@@ -88,122 +82,55 @@ private static Properties getMailProperties(String protocol, String host, String
         return properties;
     }
 
-    private static Session getSession(Properties properties) {
+    @Step(value = "mail.auth.session")
+    private Session getSession(Properties properties) {
```

Comment:
public void setSession() {
     this.session = Session.getInstance( getMailProperties(this.user,...))
}